shcDecoder/src/main/java/ca/bc/gov/shcdecoder/revocations/Utils.kt

```diff
@@ -0,0 +1,11 @@
+package ca.bc.gov.shcdecoder.revocations
+
+import ca.bc.gov.shcdecoder.cache.impl.CacheManagerImpl
+
+private const val REVOCATION_JSON_PATH = "/.well-known/crl/"
+
+fun getRevocationsUrl(iss: String, kid: String): String {
```

Comment:
we can have a private method within the Revocation ManagerImpl class.
As this is not reusable method for any other area of our code